### PR TITLE
fix(tests/django): update django snapshot test

### PR DESCRIPTION
**Pull request by avara1986**
## Description
Ignore both "sqlite.query" and "postgres.query" keys in django snapshot test to avoid random error in CI:

```
At request <Request GET /test/session/snapshot >:
   At snapshot (token='tests.contrib.django.test_django_snapshots.test_safe_string_encoding'):
    - Directory: /snapshots
    - CI mode: 1
    - Trace File: /snapshots/tests.contrib.django.test_django_snapshots.test_safe_string_encoding.json
    - Stats File: /snapshots/tests.contrib.django.test_django_snapshots.test_safe_string_encoding_tracestats.json
    At compare of 1 expected trace(s) to 400 received trace(s):
Received unmatched traces: 'sqlite.query','sqlite.query','sqlite.query','sqlite.query',','postgres.query','postgres.query','postgres.query',.......
```

## Reviewer Checklist
- [ ] Title is accurate.
- [ ] Description motivates each change.
- [ ] No unnecessary changes were introduced in this PR.
- [ ] PR cannot be broken up into smaller PRs.
- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Release note has been added for fixes and features, or else `changelog/no-changelog` label added.
- [ ] All relevant GitHub issues are correctly linked.
- [ ] Backports are identified and tagged with Mergifyio.
- [ ] Add to milestone.

### CODE REVIEW

**Review by gnufede — DISMISSED** [review]

**Review by juanjux — DISMISSED** [review]

**Review by juanjux — APPROVED** [review]

**Review by gnufede — DISMISSED** [review]

**Review by vdeturckheim — DISMISSED** [review]

**Review by juanjux — APPROVED** [review]

File: tests/contrib/django/test_django_snapshots.py [hunk 1]
```diff
@@ -98,16 +98,20 @@ def test_middleware_trace_partial_based_view(client):
...
-        "": django.VERSION >= (2, 2),
-    }
-)
-def test_safe_string_encoding(client):
-    assert client.get("/safe-template/").status_code == 200
+def test_safe_string_encoding(client, snapshot_context):
+    """test_safe_string_encoding.
+    If we use @snapshot decorator in a Django snapshot test, the first test adds DB creation traces
+    """
+    client.get("/safe-template/")
```
Comment (brettlangdon) [inline tests/contrib/django/test_django_snapshots.py:105]:
```suggestion
```

?

Comment (brettlangdon) [reply tests/contrib/django/test_django_snapshots.py:105]: Do we need this initial call?

**Review by brettlangdon — COMMENTED** [review]
do we not need this change for the other usages of `@snapshot` in this test module?

File: tests/contrib/django/test_django_snapshots.py (hunk 1 above)
Comment (avara1986) [reply tests/contrib/django/test_django_snapshots.py:105]: yes, we need it :) 

Without this line:
```
At request <Request GET /test/session/snapshot >:
   At snapshot (token='tests.contrib.django.test_django_snapshots.test_safe_string_encoding'):
    - Directory: /snapshots
    - CI mode: 0
    - Trace File: /snapshots/tests.contrib.django.test_django_snapshots.test_safe_string_encoding.json
    - Stats File: /snapshots/tests.contrib.django.test_django_snapshots.test_safe_string_encoding_tracestats.json
    At compare of 1 expected trace(s) to 1 received trace(s):
     At trace 'django.request' (33 spans):
Received more spans (35) than expected (33). Received unmatched spans: 'sqlite.query', 'django.cache'
```

**Comment by avara1986** [comment]
> do we not need this change for the other usages of `@snapshot` in this test module?

I saw the problem in tests with decorator `@pytest.mark.django_db`

**Review by juanjux — DISMISSED** [review]

File: tests/contrib/django/test_django_snapshots.py (hunk 1 above)
Comment (brettlangdon) [reply tests/contrib/django/test_django_snapshots.py:105]: can we add a comment here on why we have this extra call?

it is also likely going to be an issue if someone tries to do `pytest ... -k test_psycopg_query_default`, the initial database setup/migration will run and that test will fail (e.g. this fix/extra call will only work if this test is called first).

I am ok with it for now if we can add a comment that that is why we are doing that and we can come back later to try and solve the issue.

Comment (avara1986) [reply tests/contrib/django/test_django_snapshots.py:107]: `pytest ... -k test_psycopg_query_default` doesn't need a extra call such as test_safe_string_encoding. With these changes ~~I think~~  I hope  both tests are idempotent.

Docstring updated :+1:

**Review by juanjux — APPROVED** [review]

**Review by gnufede — APPROVED** [review]

**Pull request merged**